docs/structure.md

```diff
@@ -40,16 +40,23 @@ Contains several projects for generating and building projections from the Windo
 
 Contains various testing-related projects:
 
-- [`TestComponentCSharp`](../src/Tests/TestComponentCSharp): This is an implementation of a WinRT test component, defined in class.idl and used by the UnitTest project.  To complement the general TestComponent above, the TestComponentCSharp  tests scenarios specific to the C#/WinRT language projection.
+- [`TestComponentCSharp`](../src/Tests/TestComponentCSharp): This is an implementation of a WinRT test component, defined in **class.idl** and used by the UnitTest project.  To complement the general TestComponent above, the TestComponentCSharp  tests scenarios specific to the C#/WinRT language projection.
 
 - [`UnitTest`](../src/Tests/UnitTest): Unit tests for validating the Windows SDK, WinUI, and Test projections generated above.  All pull requests should ensure that this project executes without errors.
 
 - [`HostTest`](../src/Tests/HostTest): Unit tests for WinRT.Host.dll, which provides hosting for runtime components written in C#.
 
 ## [`src/TestWinRT`](https://github.com/microsoft/TestWinRT/)
 
-C#/WinRT makes use of the standalone [TestWinRT](https://github.com/microsoft/TestWinRT/) repository for general language projection test coverage.  This repo is cloned via **get_testwinrt.cmd** into the `src` folder, so that the cswinrt.sln can resolve its reference to TestComponent.vcxproj.  The resulting `TestComponent` and `BenchmarkComponent` files are consumed by the UnitTest  and Benchmarks projects above.
+C#/WinRT makes use of the standalone [TestWinRT](https://github.com/microsoft/TestWinRT/) repository for general language projection test coverage.  This repo should be cloned into the root of the C#/WinRT repo, via **get_testwinrt.cmd**, so that **cswinrt.sln** can resolve its reference to **TestComponent.vcxproj**.  The resulting **TestComponent.dll** and **TestComponent.winmd** files are consumed by the UnitTest project above.
```

Comment:
curious about two changes here:
"is" -> "should be" implies that the dev has some responsibility, but get_testwinrt.cmd is automatically run by build.cmd
why the removal of mention of the benchmark component?

Reply:
changed the wording back to original